### PR TITLE
feat: expose `defaultValue` to `RichText` control

### PR DESCRIPTION
**Pull request by joshuawootonn**
josh/prd-3406

### CODE REVIEW

**Comment by joshuawootonn** [comment]
@apledger said that this was something that would be helpful on these buildouts.

Thought it would also be good for me to check that the publishing CI still works :)

File: packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts [hunk 1]
```diff
@@ -54,7 +54,10 @@ export function RichText<T extends ExternalRichTextV2Config>(
     type: RichTextV2ControlType,
     config: {
       mode: config.mode,
-      defaultValue: config?.mode === RichTextV2Mode.Inline ? 'Edit this text' : ipsum(3),
+      defaultValue:
+        config?.defaultValue ?? config?.mode === RichTextV2Mode.Inline
+          ? 'Edit this text'
+          : ipsum(3),
```
Comment (migueloller) [inline packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts:60]: This won't work as you expect it to

CleanShot 2023-10-10 at 18 41 55

**Review by migueloller — CHANGES_REQUESTED** [review]

File: packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts (hunk 1 above)
Comment (joshuawootonn) [reply packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts:60]: How is that possible?

Based on this order of operations chart: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table

Wouldn't it resolve as 

```
'default_value' ?? 'one' === 'one' ? 'oh_no' : 'hi'
'default_value' ?? 'oh_no' 
'default_value'
```

Comment (joshuawootonn) [reply packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts:60]: because `'default_value'` is never null or undefined.

Comment (joshuawootonn) [reply packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts]: Ok https://stackoverflow.com/a/67575538/8239711

Comment (migueloller) [reply packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts]: The higher the number in the chart, the higher the precedence so the nullish coalesce would resolve _first_ but in your comment above you resolve it _last_

**Pull request merged**